raspbian/applications/camera.md

```diff
@@ -276,7 +276,7 @@ By default, DRC is off.
 	--stats,	-st		Display image statistics
 ```
 
-Displays the exposure, analogue and digital gains, and AWB settings used.
+Force recomputation of statistics on stills capture pass. Digital gain and AWB are recomputed based on the actual capture frame statistics, rather than the preceding preview frame.
```

Comment:
Oh, change in behaviour or just more accurate description of what the option does? Are the stats also still displayed?

Reply:
I think when I wrote up the first version I guessed what it did and never came back to correct it.